src/services/ui/src/components/RHF/Section.tsx

```diff
@@ -24,16 +24,20 @@ export const RHFSection = <TFieldValues extends FieldValues>(props: {
             <FormLabel className="font-bold">{props.section.title}</FormLabel>
           </div>
         )}
-        <div className="px-8 py-4">
-          {props.section.form.map((FORM, index) => (
-            <RHFFormGroup
-              key={`rhf-form-${index}-${FORM.description}`}
-              parentId={props.formId + "_" + props.section.sectionId + "_"}
-              control={props.control}
-              form={FORM}
-            />
-          ))}
-        </div>
+        {props.section.form?.length ? (
```

Comment:
Nice, I like this solution for ensuring an empty section has no x or y padding. 👍🏻